screens/JournalScreen.tsx

```diff
@@ -23,21 +24,31 @@ export default function JournalScreen() {
   useEffect(() => {
     const fetchJournalData = async () => {
       if (!dataLoaded) {
-        const journalData = await JournalUtil.loadJournal();
```

Comment:
@pranavcd FYI that TS was complaining about this as it's an anti-pattern to create a new variable in a nested scope with the same name as something in the enclosing scope (journalData is already assigned on line 12 of JournalScreen). I fixed this in a few other places in this file by creating a unique name for the variable inside the function.